server.js

```diff
@@ -1,27 +1,77 @@
 import express from "express";
 import cors from "cors";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
+import netflixData from "./data/netflix-titles.json";
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
-const app = express();
+const app = express(); //this function call will allows us to create the whole API boilerplate
 
 // Add middlewares to enable cors and json body parsing
+// Cors makes it easier to use the API, allows API's to say where the requests come from.
+// bodyParser allows express to read json in post requests
 app.use(cors());
-app.use(express.json());
+app.use(express.json()); //allows us to read the bodies from the request
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+
+  res.send({responseMessage: "Hello! An API for netflix-titles."});
+});
+
+app.get("/shows", (req, res) => {
+  const { title, country, year, type } = req.query;
+  
+  let totalShows = [...netflixData]; // Clone
+  // let totalShows = netflixData; Share the same memory adress
+  
+  if (title) {
+    totalShows = totalShows.filter((show) => show.title.toLowerCase() === title.toLowerCase())
```

Comment:
Nice! I tested it and it works for example https://project-express-api-jfjicqhwka-lz.a.run.app/shows?title=chocolate